app/modules/individuals/models.py

```diff
@@ -730,7 +730,7 @@ def _incremented_context(cls, ctx, existing):
     def get_blocking_encounters(self):
         blocking = []
         for enc in self.encounters:
-            if not enc.current_user_has_edit_permission():
+            if not enc.current_user_has_edit_permission() and not enc.is_public():
```

Comment:
this seems to be the straightforward fix.  i did mention the fact that notifications seem to need to be dealt with regarding the public user, but: (a) that should probably be a separate ticker (given larger scope than just _merge_ notifications; and, (b) it is not a blocking/problem.